exercises/practice/pop-count/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Licensing is done on a repo level, not an exercise level.

Reply:
yet another auto-generated file.